README.md

```diff
@@ -1,26 +1,31 @@
 # hacktoberfest2022-bad-sorting-algorithms
+
```

Comment:
Please undo the styling changes in the file. You should only be adding your name to algo you implemented